### PR TITLE
Comments corrections, and improve singleton error messages

### PR DESCRIPTION
**Pull request by Dobby233Liu**
*\~Open per request in DDMC\~*
## Features
* Small corrections to some comments.
  Some uses new lines, which may not be what you want.
* Sync singleton to the latest version in tendo, and make it output more helpful error messages.
  I should probably revert this.
* Freaking messy bunch of commits.
  Yes, I call it a feature.

### CODE REVIEW

File: game/definitions.rpy [hunk 1]
```diff
@@ -3,35 +3,41 @@
...
 define persistent.steam = ("steamapps" in config.basedir.lower())
 # Change this to True to enable Developer Mode
 define config.developer = "auto"
 
 python early:
+    # Import the singleton module from the tendo package to
+    # ensure only one instance of the game is running
     import singleton
-    me = singleton.SingleInstance()
+    me = singleton.SingleInstance(error_message="Another instance of %s is already running." % config.name)
```
Comment (Dobby233Liu) [inline game/definitions.rpy:17]: I don't think config.name is even defined here gonna take a look

Comment (Dobby233Liu) [reply game/definitions.rpy:17]: Maybe??? It's referred with `renpy.config.name` in some code so idk

**Comment by GanstaKingofSA** [comment]
I'm a bit iffy with this PR, mainly because of the singleton sync and changes. My motto really on Team Salvato files is `if it isn't broken or vulnerable to security bugs, don't fix`. Also I tried running this copy of singleton in Ren'Py 6.99 which oddly ended in a error of syntax. 
On the comments, some are not really Dan leftovers rather Ren'Py specific to any project so I will have to do a re-review of my comments then to apply any other comment fixes.

**Comment by GanstaKingofSA** [comment]
Closed due to inactivity with this PR and being superseeded by newer comments.

**Pull request closed without merging**